src/components/Tooltip/index.js

```diff
@@ -91,10 +156,15 @@ function Tooltip(props) {
         if (bounds.width === 0) {
             setIsRendered(false);
         }
-        setWrapperWidth(bounds.width);
-        setWrapperHeight(bounds.height);
-        setXOffset(bounds.x);
-        setYOffset(bounds.y);
+        const t = target.current;
+        if (!t) {
+            return;
+        }
+        const betterBounds = chooseBoundingBox(t, initialMousePosition.current.x, initialMousePosition.current.y);
```

Comment:
Please add a comment here on why we need better bounding box

Reply:
Done.